documentation/installation.md

```diff
@@ -36,6 +36,10 @@ To check if the controller is working, find it in the list of kyverno pods:
 
 The [Kyverno CLI](documentation/testing-policies-cli.md) allows you to write and test policies without installing Kyverno in a Kubernetes cluster.
 
+# Pre-Requisites
+## Configure controller manager certificate signer
+When the cluster ca & key is not passed as arguments(mostly for in-cluster mode), the TLS communicate between admission webhook and api-server a certificate signer configured to issue certificate to a certificate signing request(CSR) generated by Kyverno.
+The Kubernetes controller manager provides a default implementation of a signer. To verify if it is enabled, check if the command args --cluster-signing-cert-file and --cluster-signing-key-file are passed to the controller manager with paths to your Certificate Authority’s keypair. 
```

Comment:
The support of these flags is not implemented, so we shouldn't write about them in documentation, because it makes the project inconsistent